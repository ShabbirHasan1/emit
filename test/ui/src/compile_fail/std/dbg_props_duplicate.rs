fn main() {
    emit::dbg!(a: 42, a: true);
}
