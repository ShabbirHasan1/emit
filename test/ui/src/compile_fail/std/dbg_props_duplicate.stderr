error: keys cannot be duplicated
 --> src/compile_fail/std/dbg_props_duplicate.rs:2:23
  |
2 |     emit::dbg!(a: 42, a: true);
  |                       ^^^^^^^
